pkg/proxy/kafka.go

```diff
@@ -237,6 +445,11 @@ func (k *kafkaRedirect) handleRequest(pair *connectionPair, req *kafka.RequestMe
 
 	// Write the entire raw request onto the outgoing connection
 	pair.tx.Enqueue(req.GetRaw())
+
+	// log valid response
+	record.Timestamp = time.Now().UTC().Format(time.RFC3339Nano)
+	accesslog.Log(record, accesslog.TypeResponse, accesslog.VerdictForwarded,
```

Comment:
Unlike with HTTP, the responses are parsed as well so we can log them there. See `handleResponseConnection()`